mypyc/primitives/exc_ops.py

```diff
@@ -44,21 +42,19 @@
     error_kind=ERR_FALSE)
 
 # Assert that the error indicator has been set.
-assert_err_occured_op = custom_op(
+assert_err_occurred_op = c_custom_op(
```

Comment:
This seems to be unused, so it can actually be removed.

Reply:
yeah, that's actually what I prepared to discuss today. Then I'll just simply remove this.